testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -564,7 +564,7 @@ QUnit.test("TagBox has right tag order if byKey return value in wrong order", fu
             paging: false
         })
     });
-    this.clock.tick(timeToWait * 3);
+    this.clock.tick(timeToWait * 4);
```

Comment:
Why do you change this test?

Reply:
This test now depends on load method that requires a user to wait for data loading. A timeout in the load method in this test is `timeToWait * 4` so, we must wait for this time. ByKey method works with another timeout and it was enough to wait `timetoWait * 3` seconds.